libstuff/SHTTPSManager.cpp

```diff
@@ -80,107 +62,78 @@ int SStandaloneHTTPSManager::getHTTPResponseCode(const string& methodLine) {
     return 400;
 }
 
-SStandaloneHTTPSManager::Socket* SStandaloneHTTPSManager::openSocket(const string& host, SX509* x509) {
-    // Just call the base class function but in a thread-safe way.
-    return STCPManager::openSocket(host, x509, &_listMutex);
-}
-
-void SStandaloneHTTPSManager::closeSocket(Socket* socket) {
-    // Just call the base class function but in a thread-safe way.
-    SAUTOLOCK(_listMutex);
-    STCPManager::closeSocket(socket);
-}
-
-void SStandaloneHTTPSManager::prePoll(fd_map& fdm) {
-    // Just call the base class function but in a thread-safe way.
-    SAUTOLOCK(_listMutex);
-    return STCPManager::prePoll(fdm);
-}
-
-void SStandaloneHTTPSManager::postPoll(fd_map& fdm, uint64_t& nextActivity) {
-    list<SStandaloneHTTPSManager::Transaction*> completedRequests;
-    map<Transaction*, uint64_t> transactionTimeouts;
-    postPoll(fdm, nextActivity, completedRequests, transactionTimeouts);
-}
-
-void SStandaloneHTTPSManager::postPoll(fd_map& fdm, uint64_t& nextActivity, list<SStandaloneHTTPSManager::Transaction*>& completedRequests) {
-    map<Transaction*, uint64_t> transactionTimeouts;
-    postPoll(fdm, nextActivity, completedRequests, transactionTimeouts);
+void SStandaloneHTTPSManager::prePoll(fd_map& fdm, SStandaloneHTTPSManager::Transaction& transaction) {
```

Comment:
We now poll a single transaction instead of a list, and move the looping up to the command to poll each of it's own transactions.